Gopkg.toml

```diff
@@ -18,7 +18,7 @@
 
 [[constraint]]
   name = "github.com/3scale/3scale-go-client"
-  version = "0.0.2"
+  branch = "refactor-again"
```

Comment:
This branch no longer exists, maybe just master?

Reply:
I'll release what we have shortly and fix this up